onnxruntime/test/python/onnx_backend_test_series.py

```diff
@@ -105,7 +105,7 @@ def load_jsonc(basename: str):
     return json.loads("\n".join(lines))
 
 
-def create_backend_test(test_name=None):
+def create_backend_test(devices: list[str], test_name=None):
```

Comment:
Please write a PR description for this change.

Reply:
Updated the description.